packages/components/atoms/f-button/stories/IconButton.stories.js

```diff
@@ -0,0 +1,65 @@
+import {
+    withKnobs, select, boolean, text
+} from '@storybook/addon-knobs';
+import { withA11y } from '@storybook/addon-a11y';
+import FButton from '../src/components/Button.vue';
+import { CrossIcon, PlusIcon,  MoreVerticalIcon} from '../../../../tools/f-vue-icons/dist/f-vue-icons.js';
+
+export default {
+    title: 'Components/Atoms/f-button',
+    decorators: [withKnobs, withA11y]
+};
+
+export const IconButtonComponent = () => ({
+    components: {
+        FButton,
+        CrossIcon,
+        PlusIcon,
+        MoreVerticalIcon
+    },
+    props: {
+        buttonType: {
+            default: select('Button Type', ['primary', 'secondary', 'outline', 'ghost', 'link'], 'primary')
+        },
+        buttonSize: {
+            default: select('Button Size', ['xsmall', 'small', 'medium', 'large'], 'medium')
+        },
+        actionType: {
+            default: select('Action Type', ['button', 'submit', 'reset'], 'button')
+        },
+        href: {
+            default: text('href', '')
+        }
+    },
+    template: `
+    <div style="display: flex; width: 300px; justify-content: space-between">
```

Comment:
I'd be tempted to pull these out and assign the styles to a class defined in [our storybook config](https://github.com/justeat/fozzie-components/blob/master/packages/storybook/config/storybook/preview-head.html#L4), rather than adding inline styles.

Reply:
I thought that the styles there are universal for all the stories and here I specified some just for this specific one

Reply:
I'd move them out, just as a pattern like this could be used for spacing other things (multiple checkboxes, radio buttons, form-fields etc)